modules/apps/foundation/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/model/ConfigurationModel.java

```diff
@@ -42,6 +42,11 @@
  */
 public class ConfigurationModel implements ExtendedObjectClassDefinition {
 
+	public static final String CONFIGURATION_FOLDER = "/osgi/configs/";
```

Comment:
We can remove this constant because we have access to `PropsKeys.MODULE_FRAMEWORK_CONFIGS_DIR`.